docs/README.md

```diff
@@ -4,34 +4,38 @@ legacyUrl: https://docs.ipfs.io/
 description: The home page for developer documentation for IPFS, the InterPlanetary File System.
 ---
 
-# IPFS Documentation
+# Educate yourself
 
-**Welcome to IPFS!** We're glad you're here, and we hope this documentation site will help you make the decentralized Internet a better place for humanity. Whether you’re just learning about IPFS or are looking for detailed reference information, here's where to start.
+<p style="font-size:1.65rem; font-weight: 100; line-height: 2rem; margin-top: -15px;">Whether you're a tiny startup or a huge multinational company, you need to learn how to protect yourself and your users from the dangers of a centralized internet. This website contains everything you need to start using the InterPlanetary File System and help future-proof yourself.</p>
 
-IPFS docs are organized into the following main categories:
+## Host a static website on IPFS
 
-## Install IPFS
+Protect your site from a single point of failure by importing it to the IPFS network, and benefit from decentralized infrastructure. If you're new to IPFS, this is an excellent place to start learning how to build on the decentralized web. [Start the tutorial →](/how-to/host-single-page-site/)
 
-[Here's your guide](/install/) if you just want to install IPFS and get started right away!
+## Community projects
 
-## Concepts
+Are you looking for inspiration? You can find a huge collection of awesome community projects over at Awesome IPFS. [Go there now →](https://awesome.ipfs.io/)
 
-[Check out these key IPFS concepts](/concepts/) if you're new to IPFS or the distributed web in general. Because IPFS is a system that hopes to change how we use the Internet, it comes with many new concepts! This section aims to get you up to speed quickly.
+![The Awesome IPFS homepage.](./images/awesome-ipfs.png)
 
-## How-tos
+## Useful tools
 
-[Our how-tos and tutorials](/how-to/) offer more in-depth advice and information on IPFS specifics from customizing your install to managing files, working with peers, building apps, and more.
+Build your apps and services quicker with some help from these tools built for IPFS.
```

Comment:
```suggestion
These tools can help you use and build on IPFS more quickly and efficiently — give them a try today!
```

We may want to tweak this section a little bit sometime later, but this is a good start. However, let's not limit these to folks who are just developing (hence the change in language) ... people who just want to share files or play with the network can benefit from companion or desktop, too.